assignments/prototypes.js

```diff
@@ -32,6 +51,16 @@
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
 */
+
+function Humanoid(attributes) { 
+GameObject.call(this, Humanoid);
+this.team = attributes.team; 
+this.weapons = attributes.weapons;
+this.language = attributes.language;
+}
+  Humanoid.prototype.destroy = function() {
+  console.log(`${this.name} offers a greeting in ${this.language}.`)
```

Comment:
you're doing really well so far Steven, just be sure to read the instruction here a little more carefully -- remember that Humanoid should `inherit` the destroy method from GameObject through CharacterStats, and takeDamage from CharacterStats. reference the codepen demo and the training kit for a reminder on how this should work! then realize you need to do the same thing to give CharacterStats all the added methods from GameObject as well to complete the inheritance chain down to here!